Clarify relationship between crowdloan campaigns and parachain auctions (#4270)
w3f/polkadot-wiki: docs/learn/learn-crowdloans.md

:::

## Crowdloan Campaigns vs Parachain Auctions

It is important to recognize that crowdloan campaigns are seeking parachain slots via a parachain
auction, however these auction slots can also be won without the need for a crowdloan. To reiterate,
crowdloans require auctions but auctions do not require crowdloans. Let's take a look at a couple

> potential scenarios involving each path.

Reviewer: ```suggestion
It is important to recognize that starting a crowdloan campaign is **optional** for participating in a parachain slot 
auction. The parachain slot auction can also be won directly through self-funding without community involvement. To reiterate,
crowdloan campaigns are just one of the means to win auctions, which allow the community to participate in a trustless and permissionless way. 
```

Party A is working on a project that hopes to gain a parachain slot on
{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} but they don't have the means to
self-fund their bidding efforts in order to win the parachain auction. Party A has the ability to
start a new crowdloan campaign to help their chances of successfully landing a slot. This will allow
Party A to compete with projects that may have access to greater capital, granted the project has

> enough public appeal.

Reviewer: ```suggestion
Let's look at a scenario where Project A is hoping to gain a parachain slot on
{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }}, but they don't have enough tokens to
bid directly to win the parachain auction. Project A could benefit from
starting a new crowdloan campaign to help secure a parachian slot. Crowdloans are trustless and are supported natively on Polkadot, allowing the community to bond their tokens on Project A's behalf for the entire parachain lease duration. This will allow
Project A to compete with projects that may have access to greater capital, given the project has
sufficient community support. In return, the community contributors are rewarded by the projects that win the parachain slot, which would typically compensate for the opportunity cost of bonding their tokens for the lease duration. 
```

On the other hand, Party B also hopes to secure a parachain slot for their project. Party B is more
established and has access to capital to self-fund their attempt at winning a parachain auction.
Party B is not relying on any public funding/staking (at least via the crowdfunding mechanism) so

> they will have to determine how much funding they are able to allocate towards winning a slot.

Reviewer: ```suggestion
On the other hand, let's say Project B, which is more
established, and has access to capital, is hoping to secure a parachain slot through self-funding.
Project B is not relying on any community funding (at least via the crowdloan mechanism), so
they will have to determine how much funding they can allocate towards winning a slot.
```

Party B is in control of how much they are ultimately willing to contribute towards their goal of
gaining a parachain slot. This means even if there is little public interest, they can self-fund the
project if they have the means. In contrast, Party A may have access to very little capital but the
project could gain mass popularity amongst the public. Party A could end up overcoming the deep

> pockets of Party B via a successful crowdloan.

Reviewer: ```suggestion
Project B is in full control of how much they are ultimately willing to contribute towards their goal of
gaining a parachain slot. Project B need not work on coming up with a rewards model for community contributors like Project A. In contrast, crowdloan campaigns are beneficial for projects that have access to limited capital but have strong community support. They are also beneficial for projects that are able to successfully bid to win the auction with self-funding, but are looking for a mechanism to bootstrap their community and get noticed by the key actors in the ecosystem.
```

It is publicly visible (on-chain) whether or not a party that is bidding for a parachain auction
slot is self-funded or participating in a crowdloan. More technical details regarding the process

> for executing a crowdloan campaign are provided below.

Reviewer: ```suggestion
It is publicly visible on-chain whether or not a project is bidding directly or through a crowdloan campaign. More details regarding the process of creating and executing a crowdloan campaign are provided below.
```
